core/templates/domain/classifier/AnswerClassificationResultObjectFactory.ts

```diff
@@ -25,15 +25,22 @@ import { Outcome } from 'domain/exploration/OutcomeObjectFactory';
 export class AnswerClassificationResult {
   outcome: Outcome;
   answerGroupIndex: number;
-  ruleIndex: number;
   classificationCategorization: string;
 
+  // Note: rules used to be stored in a list of Rules, but have been moved to a
```

Comment:
I'm not sure you need this stuff. If you think about it from the point of view of a future contributor, they won't even see ruleIndex anyway, so this docstring isn't very helpful?

Also you say "ruleIndex has been deprecated and will eventually be deleted in a future PR". I don't see any ruleIndex here. Shouldn't you just delete it completely in this PR? The comment here would make more sense in places where there still is a ruleIndex, but it would probably be better to just get rid of it altogether if we are not using it any more, to avoid confusion.

Reply:
I agree, removed this docstr